library/scrollphathd/api/http.py

```diff
@@ -0,0 +1,79 @@
+import scrollphathd
+from scrollphathd.fonts import font3x5
+
+try:
+    from queue import Queue
+except ImportError:
+    from Queue import Queue
+
+from .action import Action
+from .stoppablethread import StoppableThread
+from http import HTTPStatus
```

Comment:
This is `httplib` in Python 2.x so you'll have to use something like:

```python
try:
    from http import HTTPStatus
except ImportError:
    import httplib as HTTPStatus
```

Reply:
Drat, this doesn't work since `httplib` doesn't support `httplib.OK.value` but just `httplib.OK`

Reply:
Okay, replacing all `HTTPStatus.X.value` with `http_status.X` and perhaps using:

```python
try:
    import http.client as http_status
except ImportError:
    import httplib as http_status
```
Seems to work!

Reply:
yeah, I had a feeling this would need to change.

Reply:
Hmm, when I put this in pycharm flags up all refs to OK etc as "cannot find reference to OK in client.py"? Did you mean to do "http.client as http_status" or "http.HTTPStatus as http_status"?

(also, this is largely why I can mostly code this blind - PyCharm uses pylint which is good at this sort of thing)